src/components/MessageAll/MessageAll.jsx

```diff
@@ -109,7 +109,7 @@ function MessageAll() {
             height: 300,
             marginLeft: 10,
             marginRight: 10,
-          }}
+          }}d
```

Comment:
I'm confused about the "d" on this line.  Typo?

Reply:
Weird, I pulled everything down last night, must have been a merge that I didn't realize. I'll get that fixed nice catch^

Reply:
d removed and pushed up the new version with the resolved conflicts! :D